api/src/main/java/org/openmrs/module/custombranding/db/hibernate/HibernateCssFileDAO.java

```
@@ -0,0 +1,120 @@
+		this.sessionFactory = sessionFactory;
+	}
+
+	@Override
+	public CssFile getCssFile(Integer id) {
+		return (CssFile) sessionFactory.getCurrentSession().get(CssFile.class, id);
+	}
+
+	@Override
+	public CssFile getCssFileByUuid(String uuid)  {
+		Query q = sessionFactory.getCurrentSession().createQuery("from CssFile f where f.uuid = :uuid");
+		return (CssFile) q.setString("uuid", uuid).uniqueResult();
+	}
+
+	@Override
+	public CssFile getCssFileByName(String name)  {
+		Query q = sessionFactory.getCurrentSession().createQuery("from CssFile f where f.name = :name");
+		return (CssFile) q.setString("name", name).uniqueResult();
+	}
+
+    @Override
+    public CssFile getCssFileByNameAndPath(String nameAndPath)  {
+        Query q = sessionFactory.getCurrentSession().createQuery("from CssFile f where f.nameAndPath = :nameAndPath");
+        return (CssFile) q.setString("nameAndPath", nameAndPath).uniqueResult();
+    }
+
+	@Override
+	public CssFile saveCssFile(CssFile CssFile) {
+		sessionFactory.getCurrentSession().saveOrUpdate(CssFile);
+		return CssFile;
+	}
+
+	@Override
+	public CssFile mergeCssFile(CssFile CssFile) {
+		sessionFactory.getCurrentSession().merge(CssFile); //merging two different objects with same id in session
+		return CssFile;
+	}
+
+	@Override
+	public void deleteCssFile(CssFile CssFile) {
+		sessionFactory.getCurrentSession().delete(CssFile);
+	}
+
+	@Override
+	@SuppressWarnings("unchecked")
+	public List<CssFile> getAllCssFiles() {
+		return sessionFactory.getCurrentSession().createCriteria(CssFile.class).list();
+	}
+
+	@Override
+	public OpenmrsObject getItemByUuid(Class<? extends OpenmrsObject> type, String uuid) {
+		try {
+			Criteria criteria = sessionFactory.getCurrentSession().createCriteria(type);
+			criteria.add(Expression.eq("uuid", uuid));
+			OpenmrsObject result = (OpenmrsObject) criteria.uniqueResult();
+			return result;
+		}
+		catch(Exception e) {
+			log.error("Error fetching item by uuid:" + e);
```

If it is really an error then you should rethrow the exception instead of returning null and do not log it at this point. Also prefer log.error("...", e), log.debug("..", e), etc. so that the whole stack trace is logged.